proof/crefine/RISCV64/SR_lemmas_C.thy

```diff
@@ -1787,6 +1786,19 @@ lemma obj_at_cslift_sc:
   apply fastforce
   done
 
+lemma obj_at_cslift_reply:
```

Comment:
I think Nick ended up adding this when he did the rules for `replyUnlink_ccorres` and `cancelIPC_ccorres`. So this is duplicated now, though I'm not sure where his rule ended up.